portfolio/src/main/webapp/index.html

```diff
@@ -32,7 +32,14 @@
     <script src="request.js"></script>
     <script src="calendarAPI.js"></script>
     <script src="sheetsApi.js" onload="handleClientLoad()"></script>
-    <script src="https://apis.google.com/js/api.js"></script>
+    <script src="tasksApi.js" onload="handleClientLoadTasks()"></script>
+    <script
+      async
+      defer
+      src="https://apis.google.com/js/api.js"
+      onload="this.onload=function(){};handleClientLoadTasks()"
```

Comment:
Looks like handleClientLoadTasks() will run twice, once here and once in the script tag above. Is this intentional?

Reply:
Done.